src/sentry/grouping/api.py

```diff
@@ -374,25 +373,16 @@ def get_grouping_variants_for_event(
     return rv
 
 
-def sort_grouping_variants(variants: dict[str, BaseVariant]) -> tuple[KeyedVariants, KeyedVariants]:
-    """Sort a sequence of variants into flat and hierarchical variants"""
+def sort_grouping_variants(variants: dict[str, BaseVariant]) -> KeyedVariants:
+    """Sort a sequence of variants into flat variants"""
 
     flat_variants = []
-    hierarchical_variants = []
 
     for name, variant in variants.items():
-        if name in HIERARCHICAL_VARIANTS:
-            hierarchical_variants.append((name, variant))
-        else:
-            flat_variants.append((name, variant))
+        flat_variants.append((name, variant))
```

Comment:
Maybe you're saving simplifications for a follow-up PR, but do we even need to do this? Doesn't this just get us a (materialized) copy of `variants.items()`?